billing.md

```diff
@@ -87,7 +87,7 @@ If you would like to prevent Cashier's migrations from running entirely, you may
 
     Cashier::ignoreMigrations();
 
-> {note} Stripe recommends that any column used for storing Stripe identifiers should be case-sensitive. Therefore, you should ensure the column collation for the `stripe_id` column is set to, for example, `utf8_bin` in MySQL. More info can be found [in the Stripe documentation](https://stripe.com/docs/upgrades#what-changes-does-stripe-consider-to-be-backwards-compatible).
+> {note} Stripe recommends that any column used for storing Stripe identifiers should be case-sensitive. Therefore, you should ensure the column collation for the `stripe_id` column is set to, for example, `utf8mb4_bin` in MySQL. More info can be found [in the Stripe documentation](https://stripe.com/docs/upgrades#what-changes-does-stripe-consider-to-be-backwards-compatible).
```

Comment:
Please reverse this. This is not what Stripe recommends: https://stripe.com/docs/upgrades#what-changes-does-stripe-consider-to-be-backwards-compatible